.github/workflows/test.yml

```
@@ -83,16 +83,28 @@ jobs:
           - name: "3.3"
             ruby-version: "3.3"
             runs-on: "ubuntu-latest"
+          - name: "3.3 (Thread)"
```

Could you use `+ XXX` style?

```suggestion
          - name: "3.3 + Thread"
```

---

Yes I can. I've switched to the `+ XXX` style.
Thanks for your suggestion!